reference/promise-types/measurements.markdown

```diff
@@ -20,27 +20,108 @@ bundle destined for the agent concerned, in this case `monitor`. However, it is
 not necessary to add them to the `bundlesequence`, because `cf-monitord`
 executes all bundles of type `monitor`.
 
+Information gathered by `cf-monitord` is stored in `${sys.statedir}/env_data` to
```

Comment:
So, is this description of env_data accurate?
https://docs.cfengine.com/docs/3.18/guide-introduction-directory-structure.html

```
state/env_data
This file contains a list of currently discovered classes and variable values that characterize the anomaly alert environment. They are altered by the monitor daemon.
``